### DISCUSSION TITLE
Dynamic Height for Modal

### DISCUSSION THREAD

**Discussion by raayu83** [Textualize/textual#3822] [Q&A] [answered]
Is there a Way to set a dynamic height for modals?
I want a modal to be a height of 21, but gradually shrink down to 10 if there is not enough space.

**Comment by davep** [comment]
Normally you'd use `height: auto` on whatever container you're using for the main container of that screen. However...

> I want a modal to be a height of 21, but gradually shrink down to 10 if there is not enough space.

...this part I don't quite understand. What does "not enough space" mean here?

**Reply by raayu83** [reply, marked as answer]
Well basically if the height of the terminal is very small.
Just figured it out myself, there is a min-height and max-height attribute.
